### DISCUSSION TITLE
Dasboard Live Ansicht

### DISCUSSION THREAD

**Discussion by joshia0** [tbnobody/OpenDTU#1956] [Q&A]
Hallo,

ich hatte mir so eine fertige openDTU bei Amaz....gekauft und lese damit meinen Hoymiles HMS 600 aus.
Kann man sich eigentlich auch verschiedene Live Ansichten laden ?
Hätte immer gerne gewußt, wie hoch der Monatsertrag war. Im Moment sieht man ja nur Gesamt und aktuellen Tagesertrag.
Kenne mich mit sowas leider überhaupt nicht aus.

**Comment by ghost** [comment]
Hi,
du kannst dir [openDTU_Database](https://github.com/RaBa64/OpenDTU-Database) mal ansehen.
Der der zugehörige Bruch ist[ hier](https://github.com/RaBa64/OpenDTU-Database/tree/Database) zu finden. Da sind auch Beispielbilder.
RaBa24 hat aber seit 2 Monaten nicht geupdatet. 
Leider habe ich meinen Fork von dort zerschossen; ich wollte das abkupfern. Jetzt lässt sich nur noch der Master Brunch forken, da ist die Database und die Statistiken nicht drin.
Oder du setzt dir was eigenes Externes auf. Infos [hier](https://www.opendtu.solar/firmware/) (MQTT, Web API).
